<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>72</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>325</width>
    <height>72</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>72</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>excelSort</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLineEdit" name="leSave">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLineEdit" name="leOpen">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pbOpen">
     <property name="text">
      <string>Открыть...</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pbSave">
     <property name="text">
      <string>Сохранить</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="C:/Program Files (x86)/Pyhton27/Lib/site-packages/PyQt4/examples/activeqt/webbrowser/mainwindow.qrc"/>
  <include location="C:/Program Files (x86)/Pyhton27/Lib/site-packages/PyQt4/examples/xmlpatterns/schema/schema.qrc"/>
  <include location="C:/Program Files (x86)/Pyhton27/Lib/site-packages/PyQt4/examples/widgets/stylesheet/stylesheet.qrc"/>
  <include location="C:/Program Files (x86)/Pyhton27/Lib/site-packages/PyQt4/examples/webkit/formextractor/formextractor.qrc"/>
  <include location="C:/Program Files (x86)/Pyhton27/Lib/site-packages/PyQt4/examples/animation/animatedtiles/animatedtiles.qrc"/>
  <include location="C:/Program Files (x86)/Pyhton27/Lib/site-packages/PyQt4/examples/desktop/systray/systray.qrc"/>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
